src/patternfly/components/Form/examples/Form.md

```diff
@@ -75,7 +75,7 @@ cssPrefix: pf-c-form
       {{#> form-label form-label--attribute=(concat 'for="' form--id form-group--id '"') required="true"}}
         Name
       {{/form-label}}
-      {{> form-group-label-help}}
+      {{> form-group-label-help form-group-label-help--aria-label="More information for name field" form-group-label-help--aria-describedby=(concat form--id form-group--id)}}
```

Comment:
this is unrelated, but I noticed this example and the regular horizontal example both use `form--id="form-horizontal"` - wold you mind updating this one to be unique?